include/highfive/H5Version.hpp.in

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright (c), 2020
+ *
+ *  Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE_1_0.txt or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ *
+ */
+#ifndef H5VERSION_HPP
+#define H5VERSION_HPP
+
+#define HIGHFIVE_VER_MAJOR @PROJECT_VERSION_MAJOR@
+#define HIGHFIVE_VER_MINOR @PROJECT_VERSION_MINOR@
+#define HIGHFIVE_VER_PATCH @PROJECT_VERSION_PATCH@
```

Comment:
`VERSION` for consistency? Or is this some convention?

Reply:
I follow the ticket ^^

Reply:
Using `VERSION` and `VER`… that just makes me cringe.